runtime/CSharp/runtime/CSharp/Antlr4.Runtime/Tree/ParseTreeWalker.cs

```diff
@@ -9,6 +9,16 @@ public class ParseTreeWalker
     {
         public static readonly ParseTreeWalker Default = new ParseTreeWalker();
 
+        /// <summary>
+        /// Performs a walk on the given parse tree starting at the root and going down recursively
+	    /// with depth-first search. On each node, 
+        /// <see cref="ParseTreeWalker.EnterRule(IParseTreeListener, IRuleNode)"/> is called before
+	    /// recursively walking down into child nodes, then
+	    /// <see cref="ParseTreeWalker.ExitRule(IParseTreeListener, IRuleNode)"/>
```

Comment:
why not use block comments?

Reply:
`///` line comments are the de-facto standard for C# code. Java-style `/** */` are technically allowed, but nearly non-existent in the wild.

Reply:
microsoft uses triple slashes on their [c# documentation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#an-example) so I thought of doing the same